src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Entities/Element.cs

```diff
@@ -10,9 +10,9 @@ namespace Microsoft.Sbom.Parsers.Spdx30SbomParser.Entities;
 /// Base domain class from which all other SPDX-3.0 domain classes derive.
 /// https://spdx.github.io/spdx-spec/v3.0.1/model/Core/Classes/Element/
 /// </summary>
-public abstract class Element
+public class Element
```

Comment:
`Element` should be abstract. Is there a reason we can't revert the changes in this file?

Reply:
There are some elements in a spdx 3.0 file that are of type Element. For example a NoAssertion license element has the type "Element". During parsing and deserialization I found that it is not allowed to deserialize into an abstract type, therefore this change had to be made

Reply:
I'd consider that a bug in the SPDX spec, but we're stuck with it ☹️